.github/workflows/publish.yml

```diff
@@ -23,13 +23,6 @@ jobs:
       - 
         name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v1
-      - 
-        name: Login to GitHub Container Registry
-        uses: docker/login-action@v1 
-        with:
-          registry: ghcr.io
-          username: ${{ secrets.CLOUDY_USERNAME }}
-          password: ${{ secrets.CLOUDY_TOKEN }}
```

Comment:
From yesterday's discussion, we are going to need to revert this change so that we can push the image to ghcr. We should update the secret names to `GH_USERNAME` and `GH_TOKEN`. I've asked Adam to create those secrets on the repo.

Reply:
@jordan-rash